### PR TITLE
re-added control over node resources

### PR DESCRIPTION
**Pull request by raffaelespazzoli**
#### What does this PR do?
re-added control over node resources with the new inventory variables

#### How should this be manually tested?
run the playbook and check that nodes have an allocatable that is different than the node size.

#### Is there a relevant Issue open for this?
no

#### Who would you like to review this?
cc: @redhat-cop/casl

### CODE REVIEW

**Comment by oybed** [comment]
@raffaelespazzoli this looks like a good change for other providers as well. Can you please look to see if you can update OSP, AWS, etc. ?

File: inventory/sample.gcp.example.com.d/inventory/group_vars/OSEv3.yml [hunk 1]
```diff
@@ -35,6 +35,32 @@ openshift_enable_service_catalog: false
 openshift_hosted_prometheus_deploy: false
 openshift_cfme_install_app: false
 
+openshift_node_groups:
+  - name: node-config-master 
+    labels:
+      - 'node-role.kubernetes.io/master=true'
```
Comment (oybed) [inline inventory/sample.gcp.example.com.d/inventory/group_vars/OSEv3.yml]: Also, to be super-picky - some of this yaml is off on the indentation. I believe it works as-is, but for readability please check these lines (41, 49 & 57) compared to lines 43-46, 51-54 and 59-62.

File: inventory/sample.byo.example.com.d/inventory/group_vars/OSEv3.yml [hunk 2]
```diff
@@ -51,3 +51,33 @@ openshift_node_open_ports:
     port: "1936/tcp"
   - service: "prometheus node exporter"
     port: "9100/tcp"
+<<<<<<< HEAD
```
Comment (oybed) [inline inventory/sample.byo.example.com.d/inventory/group_vars/OSEv3.yml]: Merge issues in this file - can you please fix?

**Comment by raffaelespazzoli** [comment]
everything should be fine now

**Pull request merged**